.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - prepare
     uses: ./.github/workflows/test.yml
     with:
-      action-matrix: '["lint", "install", "install --upgrade"]'
+      action-matrix: '["lint-and-install", "install --upgrade"]'
```

Comment:
Lgtm, for my experience this might confuse contributors though.
99% of `lint` fails are the missing version bump, and then the `install` still pass.

Reply:
should still pass on the upgrade... just seems like a waste to do 2 installs if the version is off